rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,139 @@
+
+#### DOI Versioning
+Most DOI repositories appear to provide support for versioning (i.e. multiple versions of a single citation available e.g. via a drop-down selection),
+the ability to store accompanying files in the DOI repository (this would be useful for storing versions of manifest and metadata tsv files, etc.),
+and the ability to store URL references (useful for linking back to a page in the Data Browser).
+Examining how both Figshare and Zenodo do DOIs, as the group minting DOIs controls everything after the prefix it is possible to have a main DOI which points to the most recent version and then versioned URLs which point to specific versions.
+
+#### Data Browser
+The Data Browser would contain a reference to the base DOI for each project. Clicking on the DOI link would redirect the user to the (external) DOI repository where the user could see all the versions of that project and download the associated manifest and metadata.tsv files for a specific version. 
+
+#### DOI Creation/Update Process
+Embedding a DOI in the metadata (e.g. biostudies_doi) during ingest would provide per-project citability.
+Provided the DOI repository supports versioning the metadata DOI field could be updated automatically by Ingest to a new version whenever a project update is processed.
+Using a DOI may be the simplest way to provide versioned views of a project.
+
+The creation/update process would perform the following steps:
+  - Ingest creates a new DOI (new project) or a new version of an existing DOI (updated project) when the submission is deemed “complete”.
+  - Upload the corresponding metadata tsv and manifest file for this version of the project to the DOI repository (need to wait for these to be generated).
+  - Upload the matrix output file(s) for this version of the project to the DOI repository.
+  - Update a project description to the DOI repository (e.g. what’s in this version?).
+  - Create a link in the DOI repository back to the project details page in the Data Browser.
+
+### Acceptance Criteria [optional]
+
+*Acceptance criteria are the conditions that a RFC must satisfy to be accepted by users or other stakeholders.* 
+
+### Unresolved Questions
+
+Must a data citation provide an *immutable* view of a project? Or is acceptable for any of the following to undergo additions, updates or deletions over time?
+  - a project's primary data
+  - a project's secondary analysis outputs
+  - a project's expression matrix outputs, generated by the Matrix Service
+  - a project's metadata
+  
+If an *immutable* view of the cited data is a requirement, is this technically feasible before a full implementation of support for versioned files (i.e. the AUDR RFC)?
+
+For Phase 1 must a data citation provide a DOI or is a "stable URL" sufficient?
+
+For Phase 2 & 3 does the Data Browser need to provide a view of the cited data on which further faceted searches can be performed?
```

Comment:
What is meant by this?

Reply:
The cited data (e.g. a Data Release) may contain some files that are older than the latest version. Currently the Data Browser indexes/displays only the latest version of all files. The cited version could be contained in a stored manifest or DSS collection. We would always provide a means to download the cited data via the HCA CLI. This question is asking whether the Data Browser needs to show a "versioned view" of data as well as the latest view and then allow the user to do faceted searches on the versioned view? e.g. The Data Release becomes an additional facet which the users can select (e.g. "latest data" vs "Data Release Q2-2020" vs "Data Release Q3-2020", etc).

Reply:
Added clarifying text - OK to resolve this?